src/lib/comms/include/sol-network.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define SOL_INET_ADDR_STRLEN 48
 
+#ifndef AF_BT_IOTIVITY
+    #define AF_BT_IOTIVITY 31
```

Comment:
This is strange, could you elaborate?

How are we expected to make this work in Linux?